modules/unsupported/mongodb/src/main/java/org/geotools/data/mongodb/FilterToMongo.java

```diff
@@ -403,9 +403,14 @@ public Object visit(BBOX filter, Object extraData) {
         Object e1 = filter.getExpression1().accept(this, Geometry.class);
 
         Envelope envelope = filter.getExpression2().evaluate(null, Envelope.class);
-
-        DBObject dbo = BasicDBObjectBuilder.start().push("$geoIntersects")
-                .add("$geometry", geometryBuilder.toObject(envelope)).get();
+                
+        DBObject geometryDBObject = geometryBuilder.toObject(envelope);
+        addCrsToGeometryDBObject(geometryDBObject);
+        DBObject dbo = BasicDBObjectBuilder
+        		.start()
+        		.push("$geoIntersects")
+                .add("$geometry", geometryDBObject)
+                .get();
```

Comment:
@newmanw hand-formatting will always attract attention, but is in my view acceptable when formatting a long method call chain because such a chain can be like a Domain Specific Language. Similarly, inline XML, SQL, or WKT can benefit from hand-formatting.

@newmanw are you using Eclipse? The project Eclipse formatter configuration supports comment tags to turn formatting off and on. I find these most useful when hand-formatting code:
```
    // @formatter:off
   [hand formatted code goes here]
    // @formatter:on
```